core/src/main/scala/org/apache/spark/internal/config/Worker.scala

```diff
@@ -20,6 +20,17 @@ package org.apache.spark.internal.config
 import java.util.concurrent.TimeUnit
 
 private[spark] object Worker {
+  val SPARK_WORKER_PREFIX = "spark.worker"
+
+  val SPARK_WORKER_RESOURCE_FILE =
+    ConfigBuilder("spark.worker.resourcesFile")
+    .internal()
+    .doc("Path to a file containing the resources allocated to the worker. " +
+      "The file should be formatted as a JSON array of ResourceInformation objects. " +
```

Comment:
I don't think this is true anymore, I think the json has to be formatted like a ResourceAllocation

Reply:
Thanks for catching this.